src/librekey/key_store_g10.c

```diff
@@ -49,6 +56,58 @@ struct s_exp {
     DYNARRAY(s_exp_t, sub_s_exp);
 };
 
+typedef struct format_info {
+    pgp_symm_alg_t    cipher;
+    pgp_cipher_mode_t cipher_mode;
+    pgp_hash_alg_t    hash_alg;
+    const char *      botan_cipher_name;
+    const char *      g10_type;
+    size_t            iv_size;
+} format_info;
+
+static const format_info formats[] = {{PGP_SA_AES_128,
+                                       PGP_CIPHER_MODE_CBC,
+                                       PGP_HASH_SHA1,
+                                       "AES-128/CBC",
```

Comment:
Just for your information, by default botan has PKCS7 padding and for case `disable padding` I should use `AES-128/CBC/NoPadding`.